wandb/sdk/service/streams.py

```diff
@@ -285,6 +288,8 @@ def _finish_all(self, streams: Dict[str, StreamRecord], exit_code: int) -> None:
         # and jupyter is disabled if at least single stream's setting set `_jupyter` to false
         exit_handles = []
         for stream in streams.values():
```

Comment:
nit: what do you think of making streams some iterable class that will return only the valid streams?

Reply:
Added a staticmethod for started streams